src/providers/test-report-validator/test-report-validator.ts

```diff
@@ -32,6 +34,11 @@ export class TestReportValidatorProvider {
 
   public isTestReportValid(data: object, category: TestCategory): boolean {
     switch (category) {
+      case TestCategory.EUAM2:
```

Comment:
These cases already exist at the bottom of the switch

One of them probably should be mod1, but please place them after each other

Reply:
Looks like there are duplicate methods I need to look into

Reply:
please do

Reply:
merged duplicates and corrected logic to match test report page